scripts/enum/item.lua

```diff
@@ -484,13 +496,44 @@ xi.item =
     GHELSBA_CHEST_KEY                   = 1024,
     PALBOROUGH_CHEST_KEY                = 1025,
     GIDDEUS_CHEST_KEY                   = 1026,
+    RANPERRE_CHEST_KEY                  = 1027,
+    DANGRUF_CHEST_KEY                   = 1028,
+    HORUTOTO_CHEST_KEY                  = 1029,
+    ORDELLE_CHEST_KEY                   = 1030,
+    GUSGEN_CHEST_KEY                    = 1031,
+    SHAKHRAMI_CHEST_KEY                 = 1032,
+    DAVOI_CHEST_KEY                     = 1033,
+    BEADEAUX_CHEST_KEY                  = 1034,
+    OZTROJA_CHEST_KEY                   = 1035,
+    DELKFUTT_CHEST_KEY                  = 1036,
+    FEIYIN_CHEST_KEY                    = 1037,
+    ZVAHL_CHEST_KEY                     = 1038,
+    ELDIME_CHEST_KEY                    = 1039,
+    NEST_CHEST_KEY                      = 1040,
+    GARLAIGE_CHEST_KEY                  = 1041,
     DAVOI_COFFER_KEY                    = 1042,
     BEADEAUX_COFFER_KEY                 = 1043,
     OZTROJA_COFFER_KEY                  = 1044,
+    NEST_COFFER_KEY                     = 1045,
+    ELDIEME_COFFER_KEY                   = 1046,
```

Comment:
Align this properly please

Reply:
Thanks for checking. Addressed
https://github.com/LandSandBoat/server/compare/7834d0dabb272fee24f052abf6672e8dc853f8a3..febeca5d1179fc3f2391c217fc331d337148d2bb